cinema/models.py

```diff
@@ -35,12 +40,19 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(instance: "Movie", filename: str) -> pathlib.Path:
+    ext = os.path.splitext(filename)[1]
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}{ext}"
+    return os.path.join("movies/", filename)
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(upload_to="movie_image_path", null=True)
```

Comment:
Code Efficiency: You have used `models.ImageField` with `upload_to` parameter as a string. You should use it as a callable that takes two arguments `instance` and `filename`, and returns a Unix-style pathname. For example: `image = models.ImageField(upload_to=movie_image_path, null=True)`